frontend/src/components/CalcSpectrum.tsx

```diff
@@ -242,107 +305,119 @@ export const CalcSpectrum: React.FC = () => {
     />
   );
   return (
-    <form onSubmit={handleSubmit(onSubmit)}>
-      {error ? <ErrorAlert message={error} /> : null}
-      <Grid container spacing={2}>
-        <Grid item xs={12} sm={8} md={5} lg={4}>
-          <Grid container spacing={3}>
-            <Grid item xs={12} sm={8} md={5} lg={5}>
-              <Database control={control}></Database>
-            </Grid>
-            <Grid item xs={12} sm={8} md={5} lg={6}>
-              <Mode control={control} />
-            </Grid>
-            <Grid item xs={12}>
-              <WavenumberRangeSlider
-                minRange={500}
-                maxRange={10000}
-                control={control}
-                setValue={setValue}
-              />
-            </Grid>
-
-            <Grid item sm={8} lg={4}>
-              <TGas control={control} />
-            </Grid>
+    <div>
```

Comment:
Instead of adding an extra node to the DOM, we could change this to a React Fragment: https://reactjs.org/docs/fragments.html

A syntactic sugar in React would be to use empty brackets for it:
```suggestion
    <>
```